pkgdb/include/flox/resolver/lockfile.hh

```diff
@@ -237,6 +237,12 @@ struct LockfileRaw
   void
   clear();
 
+  void
+  load_from_content( const nlohmann::json & jfrom );
+
+  void
+  from_v1_content( const nlohmann::json & jfrom );
```

Comment:
suggestion: add a bit of documentation on these i.e. that `load_from_content` dispatches by version and why there is no `from_v0_content`